spec/models/alchemy/element_spec.rb

```diff
@@ -117,6 +117,19 @@ module Alchemy
         expect(Element.definitions.collect { |el| el['name'] }).to include('erb_element')
       end
 
+      context "with the generated, default template" do
+        let(:template_path) { 'lib/rails/generators/alchemy/install/templates/elements.yml.tt' }
```

Comment:
Please either create your own fixture file or use the elements.yml inside the dummy app `spec/dummy/config/alchemy/elements.yml`.

Using your own fixture is probably the preferred solution.